projects/packages/backup/src/js/components/Admin.js

```diff
@@ -46,6 +48,11 @@ const Admin = () => {
 				<div className="content">
 					<AdminSectionHero>
 						<Container horizontalSpacing={ 0 }>
+							{ hasConnectionError && (
+								<Col>
```

Comment:
Hi @chrisbliss18.
I had a similar issue on another PR, so I added the margin via styles: https://github.com/Automattic/jetpack/pull/26733/files#diff-5ab291fbb54c538c26bd2043a4ecf1eaba9e469be850f8795aa40360874458b1R217

Reply:
Thanks @sergeymitr! This is exactly what I needed. I just pushed up a fix using a similar technique.